central/pruning/pruning_test.go

```diff
@@ -1146,7 +1146,7 @@ func (s *PruningTestSuite) TestAlertPruning() {
 			gc.collectAlerts(privateConfig)
 
 			// Grab the actual remaining alerts and make sure they match the alerts expected to be remaining
-			remainingAlerts, err := alerts.SearchListAlerts(ctx, getAllAlerts())
+			remainingAlerts, err := alerts.SearchListAlerts(ctx, getAllAlerts(), true)
```

Comment:
I think this should take `excludeResolved` as `false` because pruning should remove resolved alerts too.